content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -32,32 +32,60 @@ Using the latest version of kubectl helps avoid unforeseen issues.
 
 1. Download the latest release with the command:
 
+    ```bash
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
     ```
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
-    ```
 
-    To download a specific version, replace the `$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)` portion of the command with the specific version.
+    {{< note >}}
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
+
+For example, to download version {{< param "fullversion" >}} on Linux, type:
+
+```bash
+curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+```
+    {{< /note >}}
+
+    **Validate the binary (optional)**
+
+    Validate the binaries using the checksum files.
 
-    For example, to download version {{< param "fullversion" >}} on Linux, type:
-    
+    To download the checksum file:
+    ```bash
+    curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
     ```
-    curl -LO https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+
+    Validate the binary against the checksum file:
+    ```bash
+    echo "$(<kubectl.sha256) kubectl" | sha256sum --check
     ```
 
-2. Make the kubectl binary executable.
+    This results in: `kubectl: OK`
 
+    If the check fails, `sha256` exits with nonzero status and prints output similar to:
+    ```bash
+    kubectl: FAILED
+    sha256sum: WARNING: 1 computed checksum did NOT match
     ```
+
+    {{< note >}}
+    Download the same version of the binary and checksum.
+    {{< /note >}}
+
+1. Make the kubectl binary executable.
+
+    ```bash
     chmod +x ./kubectl
     ```
 
-3. Move the binary in to your PATH.
+1. Move the binary in to your `PATH`.
```

Comment:
nit: This sentence could use more detail. Something like:
Move the kubectl binary to a file location on your system `PATH`.